examples/ifc/ifc.ttl

```diff
@@ -1,24 +1,24 @@
 @prefix bldg: <urn:example#> .
 @prefix brick: <https://brickschema.org/schema/Brick#> .
-@prefix brickifc: <https://brickschema.org/extension/ifc#> .
+@prefix ref: <https://brickschema.org/schema/Brick/ref#> .
 
-bldg:ifc_project a brickifc:Project ;
-    brickifc:projectID "abc" ;
-    brickifc:fileLocation "file://./building.ifc" ;
+bldg:ifc_project a ref:ifcProject ;
+    ref:ifcProjectID "abc" ;
```

Comment:
- Just curious, is it okay this to be different from ifcGlobalID?
- Where is the definition of ifcProjectID?

Reply:
Forgot to push from my local repo and sync with the copy in the Brick project. ifcProjectID is defined now